app.py

```diff
@@ -23,6 +24,14 @@ def create_app(config_name):
             graphiql=True   # for having the GraphiQL interface
         )
     )
+    app.add_url_rule(
+        '/_healthcheck',
+        view_func=GraphQLView.as_view(
```

Comment:
Not sure this healthcheck has to be a graphQL element
a regular endpoint with a JSON kind of response is fine (Though the graphQL one is also okay).

Reply:
@fidelisojeah ,All is done